pyccel/codegen/printing/fcode.py

```diff
@@ -452,28 +455,151 @@ def _print_Import(self, expr):
         return self._get_statement(code) + '\n'
 
     def _print_PythonPrint(self, expr):
+        end = '\n'
+        sep = ' '
+        code = ''
+        empty_end = FunctionCallArgument(LiteralString(''), 'end')
+        space_end = FunctionCallArgument(LiteralString(' '), 'end')
+        for f in expr.expr:
+            if f.has_keyword:
+                if f.keyword == 'sep':
+                    sep = str(f.value)
+                elif f.keyword == 'end':
+                    end = str(f.value)
```

Comment:
I think we should add an `else` section which raises an error or a warning. Currently if I do:
```
print("hello world", flush=True)
```
Then pyccel executes without complaining and I am not aware that my keyword was ignored

Reply:
I copied this part from the ccode printer. Shall i fix both of them?

Reply:
Yes please, don't forget to update the PR description after :+1: